src/renderer/components/SettingsPage.jsx

```diff
@@ -464,6 +493,53 @@ export default class SettingsPage extends React.PureComponent {
                 </HelpBlock>
             </Checkbox>);
 
+        if (process.platform !== 'darwin') {
+            if (this.state.spellCheckerURL === null || typeof this.state.spellCheckerURL === 'undefined') {
+                options.push(
+                    <Button
+                        id='editSpellcheckerURL'
+                        key='editSpellcheckerURL'
+                        onClick={() => this.setState({spellCheckerURL: '', allowSaveSpellCheckerURL: false})}
+                        bsStyle='link'
+                    >{'Use alternate dictionary URL'}</Button>,
```

Comment:
I am not sure do we mean alternate or alternative? Sorry

Reply:
alternative, thanks! I've just changed it